qubesidle/idle_watcher_window.py

```diff
@@ -56,6 +56,7 @@ def __init__(self):
         self.wait_future = None
 
         file_descriptor = self.conn.get_file_descriptor()
+        asyncio.set_event_loop(asyncio.new_event_loop())
```

Comment:
creating new loop from within a plugin is a bad idea, better to do that before in main before loading them.